engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/date/Date.java

```diff
@@ -0,0 +1,7 @@
+package org.enso.interpreter.node.expression.builtin.date;
+
+import org.enso.interpreter.dsl.BuiltinType;
+import org.enso.interpreter.node.expression.builtin.Builtin;
+
+@BuiltinType
+public class Date extends Builtin {}
```

Comment:
This is unneeded, use `@Builtin` on `EnsoDate` instead. See `EnsoFile` for a reference.

Reply:
Gone as of [a356457](https://github.com/enso-org/enso/pull/3559/commits/a356457e24c1cd44448e5a71a2887377f4c2c7c2)